django/forms/fields.py

```diff
@@ -204,6 +204,19 @@ def has_changed(self, initial, data):
         data_value = data if data is not None else ''
         return initial_value != data_value
 
+    def get_bound_field(self, form, name):
```

Comment:
It's also called `name` in `BoundField`'s constructor. Maybe I shouldn't change it then, or change all instances of it to make it consistent.